### PR TITLE
add scripts to configure RDP droplet

### PR DESCRIPTION
**Pull request by zoidy**
Ansible scripts to configure a droplet for remote desktop via RDP. Enables curation on the droplet using a graphical desktop

### CODE REVIEW

**Comment by zoidy** [comment]
Ansible playbook has been tested on a fresh droplet running Ubuntu 20.04 LTS

**Review by yhan818 — APPROVED** [review]

**Comment by zoidy** [comment]
I'm not sure why the pull request was approved and then immediately closed. Re-opening

File: ansible/rdp/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+The `--tags` are a comma separated list used to control which parts of the playbook to run. Omit to run the entire playbook. Possible values: `install-packages, create-user, setup-xrdp, setup-sshfs`
+
+
+
+# Connecting to the RDP session
+## General Instructions
+1. Start ssh tunnel
+```
+ssh -L 33389:localhost:3389 username@host
```
Comment (yhan818) [inline ansible/rdp/README.md:59]: should be local host: 33389?

**Review by yhan818 — COMMENTED** [review]
Please review README.md line 59.

File: ansible/rdp/README.md (hunk 1 above)
Comment (zoidy) [reply ansible/rdp/README.md:59]: localhost:3389 is the correct port. (it's the default port used by the RDP protocol. It's configured on line 239 of the Ansible playbook )

**Comment by zoidy** [comment]
> Please review README.md line 59.

As mentioned above, line 59 is correct (it's what I am using right now on my machine to connect). Happy to chat about it further via a call if you prefer

**Comment by yhan818** [comment]
Got it. Thank you.

**Pull request merged**